AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2469,12 +2467,6 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   __ASDisplayNodeCheckForLayoutMethodOverrides;
```

Comment:
This method is always going to return CGSizeZero now, should it throw an assert as well? Do we need the lock?